apps/site/src/app/about/getMembers.ts

```diff
@@ -1,28 +1,14 @@
-const FEED_URL = "https://docs.google.com/spreadsheets/d/";
-const SPREADSHEET_KEY = "1DWCOQBlzA3mpa2BXYXPmQrF9_-SpoPFbTdDQuCQ83hU";
-const QUERY = "pub";
-const FORMAT = "output=tsv";
-
-const dataURL = FEED_URL + SPREADSHEET_KEY + "/" + QUERY + "?" + FORMAT;
-
-const getSheetsData = async (page: string) => {
-	const response = await fetch(dataURL);
-
-	if (response.status !== 200)
-		console.warn(
-			"Error ocurred while fetching schedule data:",
-			response.statusText
-		);
-
-	const csv = await response.text();
-	for (const line of csv.split("\n")) {
-		const [key, value] = line.split("\t");
-		if (key === page) {
-			return JSON.parse(value);
-		}
-	}
-
-	return null;
-};
-
-export const getMembers = async () => await getSheetsData("members");
+import { cache } from "react";
+import { client } from "@/lib/sanity/sanityClient";
+
+export const getMembers = cache(async () => {
+	return await client
+		.fetch(
+			`*[_type == 'boardYear'][0]{corporate[]{person->{name, profilePic{asset->{url}}, socials[0]{link}}, position}, 
+                                  logistics[]{person->{name, profilePic{asset->{url}}, socials[0]{link}}, position}, 
+                                  marketing[]{person->{name, profilePic{asset->{url}}, socials[0]{link}}, position}, 
+                                  tech[]{person->{name, profilePic{asset->{url}}, socials[0]{link}}, position}}`
```

Comment:
I'm still relatively new to Sanity myself, but is there a more succinct way to get all members @alexanderl19 ?

Reply:
I was playing around with this for a while and couldn't think of a way to avoid duplicating our projection for each department. I've left a comment with a slightly cleaned up version, but it seems this might just have to stay for now.